content/actions/creating-actions/publishing-actions-in-github-marketplace.md

```diff
@@ -64,6 +64,15 @@ To remove a published action from {% data variables.product.prodname_marketplace
 1. Select **Publish this action to the {% data variables.product.prodname_marketplace %}** to remove the check from the box.
 1. Click **Update release** at the bottom of the page.
 
+## Transferring action repositories
+
+Action repositories can be moved by exporting or cloning the original repository, making a new repository on the target platform, and then pushing the code to the new repository.
+
+- A repo can be transferred and actions will continue to work because there is a redirect in place (users of the action won't have to update their workflows)
+- An action in the marketplace is linked to a repository by ID, allowing for new releases to be published under the same marketplace listing from the "new" repo
+- If an action repository is deleted, so is the marketplace listing. The unique `name` identifier becomes available again.
```

Comment:
```suggestion

Actions published on {% data variables.product.prodname_marketplace %} are linked to a repository by their unique `name` identifier, meaning you can publish new releases of an action from the transferred repository under the same {% data variables.product.prodname_marketplace %} listing. If an action repository is deleted, the {% data variables.product.prodname_marketplace %} listing is also deleted, and the unique `name` identifier becomes available.
```

A couple more changes here to meet our [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md) and add some extra clarity:

- We use variables for product names; for example, `{% data variables.product.prodname_marketplace %}` for "GitHub Marketplace". You can find our variables in the [`data/variables`](https://github.com/github/docs/tree/main/data/variables) folder 🙂
- I combined the two bullets since they are closely related, and I fleshed out the sentences to provide readers with more context.